### PR TITLE
Finish short integration tests on L1

### PR DESCRIPTION
**Pull request by barrasso**
In this PR:

- Got the tests up to date with develop 
- Fixed the `maxBorrowingPowerReached` bug
- Added `draw` test case to the flow
- Fixed `close` test case

https://github.com/Synthetixio/issues/issues/212

### CODE REVIEW

**Comment by barrasso** [comment]
Closes #212

File: test/integration/behaviors/short.behavior.js [hunk 1]
```diff
@@ -79,20 +92,45 @@ function itCanOpenAndCloseShort({ ctx }) {
...
-						CollateralShortAsOwner = CollateralShort.connect(ctx.users.owner);
-						interactionDelay = await CollateralShortAsOwner.interactionDelay();
+					const { events } = await tx.wait();
 
-						await CollateralShortAsOwner.setInteractionDelay('0');
-					});
+					const event = events.find(l => l.event === 'CollateralWithdrawn');
+					loanId = event.args.id;
+
+					loan = await CollateralStateShort.getLoan(user.address, loanId);
```
Comment (jacko125) [inline test/integration/behaviors/short.behavior.js:114]: Should there be a check here the `loan.collateralBefore` and `loan.collateralAfter` that the correct amount has been withdrawn?

**Review by jacko125 — COMMENTED** [review]
Looks good, only one comment about the test checking the values after an action.

File: test/integration/behaviors/short.behavior.js (hunk 1 above)
Comment (barrasso) [reply test/integration/behaviors/short.behavior.js:117]: Added additional checks before/after each action 👍

**Review by jacko125 — APPROVED** [review]
the tests LGTM

**Pull request merged**